package.json

```diff
@@ -31,8 +31,8 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@blackbaud/skyux": "2.46.0",
+    "@blackbaud/skyux": "2.48.3",
     "@blackbaud/skyux-builder": "1.33.0",
```

Comment:
Wanna bump this version of Builder to `1.34.1`?

Reply:
done!